Examples/Scripts/Python/ckf_tracks.py

```diff
@@ -22,313 +135,108 @@ def runCKFTracks(
-            inputParticles=inputParticles,  # the original selected particles after digitization
-            inputMeasurementParticlesMap=digiAlg.config.outputMeasurementParticlesMap,
-            filePath=str(outputDir / "performance_seeding_trees.root"),
-        )
-        s.addWriter(trackFinderPerformanceWriter)
-
-        # Estimate track parameters from seeds
-        paramEstimation = acts.examples.TrackParamsEstimationAlgorithm(
-            level=acts.logging.INFO,
-            inputSeeds=inputSeeds,
-            inputProtoTracks=inputProtoTracks,
-            inputSpacePoints=[spAlg.config.outputSpacePoints],
-            inputSourceLinks=digiCfg.outputSourceLinks,
-            outputTrackParameters="estimatedparameters",
-            outputProtoTracks="prototracks_estimated",
-            trackingGeometry=trackingGeometry,
-            magneticField=field,
-            bFieldMin=0.1 * u.T,
-            deltaRMax=100.0 * u.mm,
-            deltaRMin=10.0 * u.mm,
-            sigmaLoc0=25.0 * u.um,
-            sigmaLoc1=100.0 * u.um,
-            sigmaPhi=0.02 * u.degree,
-            sigmaTheta=0.02 * u.degree,
-            sigmaQOverP=0.1 / 1.0 * u.GeV,
-            sigmaT0=1400.0 * u.s,
-            initialVarInflation=[1, 1, 1, 1, 1, 1],
-        )
-        s.addAlgorithm(paramEstimation)
-        outputTrackParameters = paramEstimation.config.outputTrackParameters
-
-    # Setup the track finding algorithm with CKF
-    # It takes all the source links created from truth hit smearing, seeds from
-    # truth particle smearing and source link selection config
-    trackFinder = acts.examples.TrackFindingAlgorithm(
-        level=acts.logging.INFO,
-        measurementSelectorCfg=acts.MeasurementSelector.Config(
-            [(acts.GeometryIdentifier(), ([], [15.0], [10]))]
-        ),
-        inputMeasurements=digiAlg.config.outputMeasurements,
-        inputSourceLinks=digiAlg.config.outputSourceLinks,
-        inputInitialTrackParameters=outputTrackParameters,
-        outputTrajectories="trajectories",
-        findTracks=acts.examples.TrackFindingAlgorithm.makeTrackFinderFunction(
-            trackingGeometry, field
+    s = addSeeding(
+        s,
+        trackingGeometry,
+        field,
+        TruthSeedRanges(pt=(500.0 * u.MeV, None), nHits=(9, None)),
+        ParticleSmearingSigmas(pRel=0.01),  # only used by SeedingAlgorithm.TruthSmeared
+        SeedfinderConfigArg(
+            r=(None, 200 * u.mm),  # rMin=default, 33mm
+            deltaR=(1 * u.mm, 60 * u.mm),
+            collisionRegion=(-250 * u.mm, 250 * u.mm),
+            z=(-2000 * u.mm, 2000 * u.mm),
+            maxSeedsPerSpM=1,
+            sigmaScattering=50,
+            radLengthPerSeed=0.1,
+            minPt=500 * u.MeV,
+            bFieldInZ=1.99724 * u.T,
+            impactMax=3 * u.mm,
         ),
+        TrackParamsEstimationConfig(deltaR=(10.0 * u.mm, None)),
+        acts.logging.VERBOSE,
+        inputParticles="particles_initial",
```

Comment:
Hi @paulgessinger,
`seeding.py` uses input `particles_final`, while `ckf_tracks.py` uses  input `particles_initial`. Why the difference, and what's best?
I guess it should be `particles_final`, so I made that the `addSeeding` default and override it here to keep the same output for `pytest`. What do you think?
Thanks, Tim.

Reply:
@paulgessinger, any ideas on this discrepancy between `seeding.py` and `ckf_tracks.py`? Should we keep it or change to one or the other?

Reply:
We decided to keep this discrepancy in for this PR to reproduce the original root_file_hashes, but correct it in a future PR.
I'll just check if I need to change the default in `addSeeding` and can resolve this.